chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/RealWorldTestingUploadJob.java

```diff
@@ -0,0 +1,343 @@
+            HtmlEmailTemplate email = new HtmlEmailTemplate();
+            email.setStyles(getStyles());
+            email.setBody(getBody(rwts));
+            return email.build();
+        }
+
+        private String getBody(List<RealWorldTestingUpload> rwts) {
+            StringBuilder table = new StringBuilder();
+
+            table.append("<table class='blueTable'>\n");
+            table.append("    <thead>\n");
+            table.append("        <tr>\n");
+            table.append("            <th>\n");
+            table.append("                CHPL Product Number\n");
+            table.append("            </th>\n");
+            table.append("            <th>\n");
+            table.append("                Type\n");
+            table.append("            </th>\n");
+            table.append("            <th>\n");
+            table.append("                Last Checked Date\n");
+            table.append("            </th>\n");
+            table.append("            <th>\n");
+            table.append("                URL\n");
+            table.append("            </th>\n");
+            table.append("            <th>\n");
+            table.append("                Result or Errors\n");
+            table.append("            </th>\n");
+            table.append("        </tr>\n");
+            table.append("    </thead>\n");
+            table.append("    <tbody>\n");
+            int i = 1;
+            for (RealWorldTestingUpload rwt : rwts) {
+                String trClass = i % 2 == 0 ? "even" : "odd";
+
+                table.append("        <tr class=\"" + trClass + "\">\n");
+                table.append("            <td>");
+                table.append(rwt.getChplProductNumber());
+                table.append("            </td>\n");
+                table.append("            <td>");
+                table.append(rwt.getType());
+                table.append("            </td>\n");
+                table.append("            <td>");
+                table.append(rwt.getLastChecked());
+                table.append("            </td>\n");
+                table.append("            <td>");
+                table.append(rwt.getUrl());
+                table.append("            </td>\n");
+                table.append("            <td>");
+                table.append(getErrorsAsString(rwt.getErrors()));
+                table.append("            </td>\n");
+                table.append("        </tr>\n");
+                ++i;
+            }
+            table.append("    </tbody>\n");
+            table.append("</table>\n");
+
+            return table.toString();
+        }
+
+
+        private String getErrorsAsString(List<String> errors) {
+            if (errors.size() > 0) {
+                return StringUtils.join(errors, "<br/>");
+            } else {
+                return "Successfully updated.";
+            }
+        }
+
+        private String getStyles() {
```

Comment:
Would it be worth externalizing this as a property? If we need to do something for 508 purposes or whatever it might be worth it to have this as some sort of external CSS file that can be imported here